job-dsl-plugin/src/test/resources/javaposse/jobdsl/plugin/casc/SeedJobTest.yml

```diff
@@ -0,0 +1,9 @@
+jobs:
+  - >
+      multibranchPipelineJob('configuration-as-code') {
```

Comment:
Seems like the Groovy really belongs in a separate file that could be referenced by name.

Reply:
This is an option for future improvements